exchanges/interfaces.go

```diff
@@ -188,3 +189,11 @@ type MarginManagement interface {
 	futures.PNLCalculation
 	GetFuturesContractDetails(ctx context.Context, item asset.Item) ([]futures.Contract, error)
 }
+
+// LimitedScope defines a subset of the exchange interface
+type LimitedScope interface {
```

Comment:
I think it would be good to rename it to be more relevant to its limited scope. `ConfigGeneration`

Reply:
I purged it, I was being lazy, you can see why in recent update.